content/millennium/r4/other/binary.md

```diff
@@ -102,6 +117,21 @@ An `Accept` header of `application/xml` could be supplied instead, if the XML Bi
 
 <%= disclaimer %>
 
+#### Patient Authorization Request
+
+  ### Example
+
+  #### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/$autogen-ccd-if?patient=1694010
```

Comment:
Update it to closed end points 
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197198634

Reply:
Updated https://github.com/cerner/fhir.cerner.com/pull/563/commits/ab6969b15631293ec930c2aeffa90629c3eeb1ca